composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/console": "~2.3",
         "sonata-project/core-bundle": "^2.3.2",
         "sonata-project/admin-bundle": "~2.3",
-        "friendsofsymfony/user-bundle": "~1.3",
+        "friendsofsymfony/user-bundle": "~1.3 || 2.0.*@dev",
```

Comment:
Just `~1.3|~2.0`. See this: https://github.com/sonata-project/SonataUserBundle/blob/master/composer.json#L52-L53